internal/core/src/index/Utils.cpp

```diff
@@ -120,11 +120,12 @@ GetIndexTypeFromConfig(const Config& config) {
     return index_type.value();
 }
 
-std::string
+int32_t
```

Comment:
Please use the alias `IndexVersion` instead.

Reply:
done